src/globus_sdk/services/groups/client.py

```diff
@@ -1,8 +1,161 @@
+from enum import Enum
+from typing import Optional, Sequence
+
 from globus_sdk import client
 
 from .errors import GroupsAPIError
 
 
+class Role(Enum):
+
+    member = "member"
+    manager = "manager"
+    admin = "admin"
+
+
+class GroupMemberVisibility(Enum):
+
+    members = "members"
+    managers = "managers"
+
+
+class GroupVisibility(Enum):
+
+    authenticated = "authenticated"
+    private = "private"
+
+
+class RequiredSignupFields(Enum):
+    institution = "institution"
+    current_project_name = "current_project_name"
+    address = "address"
+    city = "city"
+    state = "state"
+    country = "country"
+    address1 = "address1"
+    address2 = "address2"
+    zip = "zip"
+    phone = "phone"
+    department = "department"
+    field_of_science = "field_of_science"
+
+
+class BatchMembershipActions:
+    """
+    An object used to represent a batch action on memberships of a group.
+    `Perform actions on group members
+    <https://groups.api.globus.org/redoc#operation/group_membership_post_actions_v2_groups__group_id__post>`_.
+
+    .. automethodlist:: globus_sdk.GroupsClient
+    """
+
+    def __init__(self):
+        self._payload: dict = {}
+
+    def accept_invites(self, identity_ids: Sequence[str]):
+        """
+        Accept invites for identities.  The identities must belong to
+        the identity set of authenticated user.
+        """
+        self._payload.setdefault("accept", []).extend(
+            {"identity_id": identity_id} for identity_id in identity_ids
+        )
+        return self
+
+    def add_members(self, identity_ids: Sequence[str], role: Role = Role.member):
```

Comment:
As I look at this PR, and try to weigh the basic flexibility vs safety tension that surrounds use of strict types (enums, helper objects) over built-in, more generic types (str, dict), this `def` strikes me as possibly a place where we could strike a balance.

The helper object could require the enums, and make no allowances for strings or other objects, but then the client methods could accept `Union[<helper_object>, Dict]`.

That way, you can always call `to_dict()` and add explicit values to a payload, but the helper itself will have guardrails for the 99% use-case.

What do you think?